### DISCUSSION TITLE
add ingame support to directly add featured artist songs

### DISCUSSION THREAD

**Discussion by Saria0108** [ppy/osu#21840] [Ideas] [answered]
add ingame support to directly add featured artist songs

adding this would be a little quality of life thing as well as incentivizing mappers to use featured artist songs, i think adding the the featured artist page ingame and hitting the "download beatmap template" could automatically load the template in the editor, without having to download the template from the website in a browser and then directly importing the file into lazer

i think adding this wouldnt do any harm and would again just be a quality of life thing to also get more people to map featured artist songs

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/18904

> or choosing from the featured artist library

**Reply by Saria0108** [reply]
what i mean is adding the library ingame

**Reply by ghost** [reply]
yes that's what the issue is suggesting
